_meetup/16-Jan-2016.md

```diff
@@ -1,8 +1,8 @@
 ---
 layout: meetup
-title: "Meetup #15 - 16 Jan 2016"
+title: "Meetup #15 - Coming 16 Jan 2016"
```

Comment:
I also think it's overkill for now. It should have worked with the existing code base but looks like we missed something. I will look into this more maybe sometime this weekend. :ok: